### DISCUSSION TITLE
Continuous error with 1.18.1 modpacks

### DISCUSSION THREAD

**Discussion by piiiper** [FabricMC/fabric-api#2134] [Player support]
Not sure what's going on here, I'm not too familiar with anything I'm looking at, but was told this was an issue with Fabric less than the launcher or mods I was using.
[latest.log](https://github.com/FabricMC/fabric/files/8493877/latest.log)

**Comment by deirn** [comment]
Plasmo Voice is bundling Fabric API modules built for Minecraft 1.18.2 that happen to be incompatible with 1.18.1. I suggest opening an issue there.
